htdocs/compta/facture/card.php

```diff
@@ -1084,6 +1084,7 @@
 										}
 
 										if ($totalamount != 0) {
+											$i = $i > 0 ? $i-1 : 0;
```

Comment:
$i is the cursor var of the loop. Using its value outside the loop is not a good practise. May be you want to user $numlines -1 instead of $i ?